configure.ac

```diff
@@ -696,10 +696,6 @@ case $host in
            BDB_LIBS="-L$bdb_prefix/lib -ldb_cxx-4.8"
          fi
 
-         if test "x$use_sqlite" != xno && $BREW list --versions sqlite3 >/dev/null; then
```

Comment:
note: this is different from #25985 as https://github.com/fanquake/bitcoin/commit/d6d402bd2b8211e9e111acae433ca4e3cfd8d370 removed the x-prefix in the meantime